jcloud/helper.py

```diff
@@ -193,3 +195,232 @@ class CustomHighlighter(ReprHighlighter):
     highlights = ReprHighlighter.highlights + [
         r"(?P<url>(grpc|grpcs)://[-0-9a-zA-Z$_+!`(),.?/;:&=%#]*)"
     ]
+
+
+# Code below was ported over from JAML class in jina; we need to utiltiy of expand_dict during the normalization process.
+
+context_regex_str = r'\${{\s[a-zA-Z0-9_]*\s}}'
```

Comment:
Please move this to a different file (e.g.- `env_helper.py`) file, and add a line that says "This code is directly taken from jina/helper.py".

Reply:
done